tests/Wrapper/VariationTest.php

```diff
@@ -205,6 +205,32 @@ public function testChildCategoriesAreProperlyBuilt(): void
         $this->assertSame('/living-room/armchairs-stools/', $wrapper->getAttributes()[1]->getValues()[0]);
     }
 
+    public function testCategoryIsSkippedIfParentCategoryIsNotVisible(): void
```

Comment:
Add a bit more context, so it is clear how a category can be "not visible".

```suggestion
    public function testCategoryIsSkippedIfParentCategoryIsNotVisibleForExportingClient(): void
```